lib/utils/host/hostusers.go

```diff
@@ -56,6 +57,15 @@ func GroupAdd(groupname string, gid string) (exitCode int, err error) {
 	if cmd.ProcessState.ExitCode() == GroupExistExit {
 		return cmd.ProcessState.ExitCode(), trace.AlreadyExists("group already exists")
 	}
+
+	if cmd.ProcessState.ExitCode() == GroupInvalidArg {
```

Comment:
Up to you, but it may be about time to `switch code := cmd.ProcessState.ExitCode(); code {`...

Reply:
Yeah I was thinking about that after I opened the PR. I'll go ahead and do that :+1: